apps/portals/adknowledgeportal/src/config/explorePageRoutes.tsx

```diff
@@ -19,7 +19,9 @@ import { targetEnablingResourcesQueryWrapperPlotNavProps } from './synapseConfig
 export const explorePageRoutes: RouteObject[] = [
   {
     path: 'Programs',
-    element: <CardContainerLogic {...programs} sql={programsSql} />,
+    element: (
+      <CardContainerLogic cardConfiguration={programs} sql={programsSql} />
+    ),
```

Comment:
Modified `CardContainerLogic` so the `CardConfiguration` props are now nested in a `cardConfiguraton` object